opm/autodiff/RateConverter.hpp

```diff
@@ -613,13 +613,16 @@ namespace Opm {
                     coeff[iw] = 1.0 / bw;
                 }
 
+                // Actual Rs and Rv:
+                double Rs = ra.rs;
```

Comment:
I could not see the difference this change can result. I mean, the new code and old code basically the same. Maybe I oversaw something here?

Reply:
Or it was the intention to have the same change as the function `calcReservoirVoidageRates`?

Reply:
Make Rs and Rv const, also in the other function.